docs/1.getting-started/6.data-fetching.md

```diff
@@ -352,3 +352,36 @@ Using `$fetch` has a number of benefits, including:
 It will handle 'smartly' making direct API calls if it's running on the server, or making a client-side call to your API if it's running on the client. (It can also handle calling third-party APIs.)
 
 Plus, it comes with convenience features including automatically parsing responses and stringifying data.
+
+When using server-side rendering, calling `$fetch` in an isomorphic code (which is executed on both server and client side) will fetch the data twice (initially on the server and again on the client during the hydration phase). `$fetch` will *not serialize* and *not transfer* the data to the client side (via the payload) and the data will be fetched again on client side.
+
+For Example:
+
+**/pages/price.vue**: Isomorphic code below executes `$fetch` twice (initially on the server, then again on the client).
+
+```ts
+<script setup lang="ts">
+  const price = $fetch('/api/price');
+</script>
+```
+
+**/server/api/product.get.ts**: Server only code below executes `$fetch` only once at the server side.
+```ts
+export default eventHandler(async (event: H3Event) => {
+  const price = $fetch('/api/price');
```

Comment:
I think we can move toward and recommend `event.$fetch` which has already some benefits such as preserving context, and also future possibilities like same request loop detects.

```suggestion
  const price = event.$fetch('/api/price');
```

Reply:
@pi0, please forgive my lack of info. This is the first time I ever heard of `event.$fetch`. Even googling returns only one page about it with a single line of info [on release notes (3.2.0)](https://nuxtjs.org/releases/)

Referencing `event.$fetch` in an example without ever giving details could be confusing. Could you please add enough details about it?

Reply:
You are right. It would be a nice idea to also add a section for the `event.$fetch` in server section.

Reply:
Thanks @pi0. I'm also looking forward to learning what `event.$fetch` is.

Reply:
You can read more in this PR: https://github.com/unjs/h3/pull/323

`event.$fetch` is bound to user request. Because of this, it can automatically proxy the context and incoming headers from SSR without need to manually pass them.